src/process.ts

```diff
@@ -13,8 +15,16 @@ nova.workspace.config.onDidChange(
     console.log("Updating ESLint executable for workspace", eslintPath);
   }
 );
++nova.workspace.config.onDidChange(
+  "apexskier.eslint.config.eslintConfigPath",
+  async () => {
```

Comment:
I don't think this needs to be async, right?
```suggestion
  () => {
```

Reply:
Nope, copy/pasta mistake.